include/bootloader/core/messages.h

```diff
@@ -20,21 +21,34 @@ typedef struct {
 } UpdateData;
 
 #define UPDATE_DATA_MESSAGE_SIZE    64
-#define UPDATE_DATA_MAX_BYTE_COUNT  56
+#define UPDATE_DATA_MAX_BYTE_COUNT  52
```

Comment:
can we make this `#define UPDATE_DATA_MAX_BYTE_COUNT (UPDATE_DATA_MESSAGE_SIZE - sizeof(UpdateData))`?

Reply:
unfortunately no because of C padding structs, unless we pack it the sizeof(updataData) is 32 instead of 12. Left a comment in the code